persistence-modules/spring-boot-persistence-5/src/main/java/com/baeldung/transactional/OrderService.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.transactional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class OrderService {
+
+    @Autowired
+    private TestOrderRepository orderRepository;
+
+    @Transactional //<-- marks this method
```

Comment:
```suggestion
    @Transactional
```
I think we should omit this, usually the guideline is to describe in the article rather than in comments